### PR TITLE
feat: vcf annotation to allow assembly param (#125)

### LINKED ISSUE #125: VRS / VCF Annotation Tool should allow for input assembly
Default for the translator is GRCh38, but we should add an option to allow for something different.

### PR DESCRIPTION
**Pull request by korikuzma**
Close #125

### CODE REVIEW

**Comment by korikuzma** [comment]
@wesleygoar I updated the annotate method to take in the assembly parameter + added test for this change.

**Review by wesleygoar — APPROVED** [review]

**Review by larrybabb — APPROVED** [review]
@korikuzma this looks good to me.

**Review by ahwagner — APPROVED** [review]

**Comment by korikuzma** [comment]
Going to hold off merging until #123 is reviewed by a code owner and merged.

**Pull request merged**
